src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -699,8 +693,15 @@ public static function createActivityFromMail($mail_id, $object_mode = false)
 		$mail = self::ItemArrayFromMail($mail_id);
 		$object = self::createNote($mail);
 
-		$object['to'] = $object['cc'];
-		unset($object['cc']);
+		if (!empty($object['cc'])) {
+			$object['to'] = array_merge($object['to'], $object['cc']);
+			unset($object['cc']);
+		}
+
+		if (!empty($object['bcc'])) {
+			$object['to'] = array_merge($object['to'], $object['bcc']);
```

Comment:
Huh? Won't this reveal the recipients addresses in the activity object?

Reply:
This is only for direct mails - they do have a single recipient by now. Also this is - AFAIK - needed for Mastodon.